package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jupiterone/graph-addigy",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
Unless something has changed, we normally don't bump this package until we do our release PR.